defaults/main.yml

```diff
@@ -31,6 +31,9 @@ jitsi_meet_stun_servers:
 # Configure nginx and jitsi-meet to listening also IPv6
 jitsi_meet_ipv6_enable: true
 
+# Shall JitsiMeet contact gravatar.com
+jitsi_meet_disable_third_party_requests: true
```

Comment:
The default Jitsi Meet value for  *third party requests* is false, the default value in role variables should respect that. You can change it?

Reply:
Funny I just found out that the variable already exists in the file, also true ..

See:
https://github.com/UdelaRInterior/ansible-role-jitsi-meet/blob/f2ccf3f08818494446a8f908a562586ea5ee404b/defaults/main.yml#L150

I will merge this two lines and set it on false.